sql/parser2/scan_test.go

```diff
@@ -40,11 +40,9 @@ func TestScanner(t *testing.T) {
 		{`>`, []int{'>'}},
 		{`>=`, []int{GREATER_EQUALS}},
 		{`=`, []int{'='}},
-		{`=>`, []int{EQUALS_GREATER}},
```

Comment:
Ah, that's it. Thanks for the explanation. LGTM.
On Jul 17, 2015 7:14 PM, "Peter Mattis" <notifications@github.com> wrote:

> In sql/parser2/scan_test.go
> <https://github.com/cockroachdb/cockroach/pull/1720#discussion_r34938879>:
>
> > @@ -40,11 +40,9 @@ func TestScanner(t *testing.T) {
> >  		{`>`, []int{'>'}},
> >  		{`>=`, []int{GREATER_EQUALS}},
> >  		{`=`, []int{'='}},
> > -		{`=>`, []int{EQUALS_GREATER}},
>
> Nope. You're probably thinking of >=. Search the postgres grammar I
> linked to. There are only 2 references to EQUALS_GREATER.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/1720/files#r34938879>.
>